tests/foreman/api/test_discoveredhost.py

```diff
@@ -127,13 +125,14 @@ def test_positive_create(self):
         """
 
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1349364)
     @tier2
     def test_positive_upload_facts(self):
         """Upload fake facts to create a discovered host
 
         @id: c1f40204-bbb0-46d0-9b60-e42f00ad1649
 
+        @BZ: 1349364, 1392919
+
         @Steps:
 
         1. POST /api/v2/discovered_hosts/facts
```

Comment:
@sghai perhaps we should add a step 2 "read the created discoveredhost", because any way in order to create the discovered host we have POST first, what do you think?